### PR TITLE
[Logs] Remove logpull refs in logpush + Add Datadog/Splunk to ToC

### PR DESCRIPTION
**Pull request by crwaters16**

### CODE REVIEW

File: products/logs/src/content/logpush/logpush-configuration-api/understanding-logpush-api.md [hunk 1]
```diff
@@ -79,8 +79,10 @@ When using Sumo Logic, you may find it helpful to have [Live Tail](https://help.
 You can specify your cloud service provider destination via the required `destination_conf` parameter.
 
 * **AWS S3**: bucket + optional directory + region + optional encryption parameter (if required by your policy); for example: `s3://bucket/[dir]?region=<region>[&sse=AES256]`
+* **Datadog**: Datadog endpoint URL + Datadog API key + optional parameters; for example: `datadog://<DATADOG-ENDPOINT-URL>?header_DD-API_KEY=<DATADOG-API-KEY>&service=<SERVICE>&host=<HOST>&ddsource=<SOURCE>`
```
Comment (crwaters16) [inline products/logs/src/content/logpush/logpush-configuration-api/understanding-logpush-api.md:82]: @soheiokamoto , I added the examples from the Enable pages we just created for Splunk and Datadog, but they're formatted differently than the other examples. Can I change the <> to [] to follow the other examples, or what do you recommend here?

Comment (soheiokamoto) [reply products/logs/src/content/logpush/logpush-configuration-api/understanding-logpush-api.md:82]: As far as I know `<>` indicates parameter value, and `[]` indicates optional value, so I don't think you should change those for these.

Comment (soheiokamoto) [reply products/logs/src/content/logpush/logpush-configuration-api/understanding-logpush-api.md:82]: On the other hand, it seems like not all of them are following the convention properly, and use `[]` for required part, so may be we should fix those.
In any case, I don't have strong opinion, as long as they are consistent throughout this docs (at least within Logs).

File: products/logs/src/content/logpush/logpush-dashboard/index.md [hunk 2]
```diff
@@ -75,8 +72,8 @@ Tips for seeing the token:
 
 </Aside>
 
-13. Next in the **Customize log** screen, select the data set in which you're interested (currently `HTTP requests` or `Spectrum events`). You can keep the default fields to include in your log or make changes. You can add or remove fields at a later time by modifying your settings in **Logs** > **Logpush** (select the wrench icon).
+10. Next in the **Customize log** screen, select the data set in which you're interested (currently `HTTP requests` or `Spectrum events`). You can keep the default fields to include in your log or make changes. You can add or remove fields at a later time by modifying your settings in **Logs** > **Logpush** (select the wrench icon).
 
-14. To finish enabling Logpush, click **Save and Start Pushing**
+11. To finish enabling Logpush, click **Save and Start Pushing**s
```
Comment (soheiokamoto) [inline products/logs/src/content/logpush/logpush-dashboard/index.md]: You probably didn't mean to have the last `s`.

**Comment by soheiokamoto** [comment]
CC: @bharatnc

File: products/logs/src/content/logpush/logpush-dashboard/index.md (hunk 2 above)
Comment (crwaters16) [reply products/logs/src/content/logpush/logpush-dashboard/index.md]: Ugh thought I caught this one - thanks Sohei

**Comment by tlozoot** [comment]
Aside from conflicts with https://github.com/cloudflare/cloudflare-docs/pull/775, looks great :)

**Review by dcpena — APPROVED** [review]

**Pull request merged**